modular_skyrat/modules/modular_items/code/psych_cart.dm

```diff
@@ -0,0 +1,12 @@
+/obj/item/cartridge/psychologist
```

Comment:
Was there any reason why this isn't a subtype (child) of the medical cartridge?

Reply:
I couldn't remember if that'll force the robot calling feature onto the cartridge when a psych doesn't exactly need the ability to call medbots if they're going to be in their office